Repositories/ProjectRepository.cs

```diff
@@ -174,10 +178,22 @@ public override async Task<Project> FindAsync(int id)
             //Execute the IQueryable to get a collection of results
             List<Project> projectResults = await queryableProjects.ToListAsync();
 
-            //Redact the user after fetching the collection from the project (no separate query needs to be executed)
-            projectResults.ForEach( project => project.User = RedactUser( project.User ) );
-
```

Comment:
Dit was door ramon gedaan in https://github.com/DigitalExcellence/dex-backend/issues/376. Dit moet vlm niet gedelete worden. Misschien zijn de changes een beetje in de war geraakt omdat Ramon vlm de pull request na jou had gemaakt.